vertx-sql-client/src/main/java/io/vertx/sqlclient/spi/DatabaseMetaData.java

```diff
@@ -0,0 +1,26 @@
+package io.vertx.sqlclient.spi;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+/**
+ * Contains static metadata about the backend database server 
+ */
+@VertxGen
+public interface DatabaseMetaData {
+  
+  /**
+   * @return The product name of the backend database server
+   */
+  public String getDatabaseProductName();
+  
+  /**
+   * @return The major version of the backend database server
+   */
+  public int getDatabaseMajorVersion();
```

Comment:
simply `majorVersion()`, no need to prefix with `database` as we are in a class prefixed `Database`